posix.mak

```diff
@@ -355,14 +381,14 @@ dlangspec.verbatim.txt : $(DMD) verbatim.ddoc dlangspec-consolidated.d
 # Git rules
 ################################################################################
 
-../%-${LATEST} :
-	git clone -b v${LATEST} --depth=1 ${GIT_HOME}/$* $@
+$(PWD)/%-${LATEST} :
+	git clone -b v${LATEST} --depth=1 ${GIT_HOME}/$(notdir $*) $@
 
-../%-${DUB_VER} :
-	git clone --depth=1 -b v${DUB_VER} ${GIT_HOME}/$* $@
+$(PWD)/%-${DUB_VER} :
```

Comment:
Furthermore this broke the clone targets, they are located at `$(PWD)/../%-${LATEST}`.